nifi-nar-bundles/nifi-poi-bundle/nifi-poi-processors/pom.xml

```diff
@@ -17,7 +17,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <properties>
-        <poi.version>5.0.0</poi.version>
+        <poi.version>5.2.2</poi.version>
```

Comment:
Is this update related to the `xmlsec` dependency?

Reply:
Sorta. 5.0.0 IIRC was bringing in the old vulnerable version.